config.py.sample

```diff
@@ -10,3 +11,22 @@ TRAC_TICKET_PREFIX = 'https://trac.chevah.com/ticket/'
 DEFAULT_GITHUB_USER = None
 
 FILE_EXTENSION = '.mediawiki'
+
+# Trac ticket Component to GitHub repository mapping.
+REPOSITORY_MAPPING = {
+    'client': 'client',
+    'commons': 'commons',
+    'trac-migration-staging': 'trac-migration-staging',
+    }
+
+# GitHub repository for Trac tickets with Component not in the mapping.
+FALLBACK_REPOSITORY = 'server'
```

Comment:
> ```
> FALLBACK_REPOSITORY = None
> or
> FALLBACK_REPOSITORY = SkipMigration()
> ```

We can exclude these repositories from the `select_tickets` method. I will add a feature to skip already-submitted tickets seen in `tickets_created.tsv`.